packages/plugin-form-builder/src/utilities/lexical/converters/indent.ts

```diff
@@ -0,0 +1,47 @@
+import type { HTMLConverter } from '../types'
+
+import { convertLexicalNodesToHTML } from '../serializeLexical'
+
+type Converter = HTMLConverter<any>
+
+type ConverterFunctionParams = Parameters<Converter['converter']>[0]
+
+async function convertFunction({
+  converters,
+  node,
+  parent,
+  submissionData,
+}: ConverterFunctionParams): Promise<string> {
+  const filteredSelfConverters = converters.filter(
+    (converter) => converter.converter !== convertFunction,
+  )
+
+  let text = await convertLexicalNodesToHTML({
+    converters: filteredSelfConverters,
+    lexicalNodes: [node],
+    parent,
+    submissionData,
+  })
+
+  const styleArray = []
+
+  if (node.indent) {
+    styleArray.push(`padding-inline-start: ${node.indent * 20}px`)
+  }
+
+  const firstTag = text.slice(0, text.indexOf('>') + 1)
+  if (styleArray.length > 0) {
+    if (firstTag.includes('style')) {
+      text = text.replace('style="', `style="${styleArray.join('; ')}`)
```

Comment:
I think it might be better to add all this directly to the paragraph/text/heading/... html converters (for both indent and align handling). That way we'd have a more performant and stable solution that doesn't rely on text replacement.

In order to handle this within those converters, we should expose helper functions that can easily be called to generate the style tags

Reply:
Okay, so instead of creating two converters, I'll create a file in the `richtext-lexical\src\utilities` folder that exports an implementation that replaces the HTML string so that I can use it in the other converter.

I'll close this PR and create a new one. Thank you.